### DISCUSSION TITLE
Segfault while executing a test input file

### DISCUSSION THREAD

**Discussion by HeejaeJu** [idaholab/moose#16827] [Q&A General] [answered]
Hello Moose team.

I'was following the tutorial for app development tutorial and contact a segmentation fault error message while executing simple_diffusion.i using peacock.
The message is as bellow.

Found executable: /Users/heejaeju/Moose/tutorial/tutorial-opt
[1]    13124 segmentation fault  ~/Moose/moose/python/peacock/peacock -i simple_diffusion.i

How can I fix it?

Thanks in advance for any help!

**Comment by GiudGiud** [comment, marked as answer]
Hello

Can you please give us more details on your environment? Is this a Linux machine or Windows Subsystem for Linux?
Does it work when you run the input file without peacock ? (/Users/heejaeju/Moose/tutorial/tutorial-opt -i simple_diffusion.i)
Does it work when you start peacock without an input file and load the input file through the GUI ?
This is simple_diffusion.i from the tests for step 1 right?

Hopefully this will give some insights.
Tagging @aeslaughter since he is the most knowledgeable about peacock. 

Best,
Guillaume

**Reply by aeslaughter** [reply]
@GiudGiud is correct, we need some information on your system and environment. Also, if you just run Peacock without any arguments, does it still fail?

**Reply by HeejaeJu** [reply]
Thank you for the reply. 

Yes, this is the simple_diffuion.i.
My environment is Macintosh(OS : BigSur). Execution without peacock works well.

And I found that the same error message is printed when I try to start peacock.exe.
In addition, double clicking the peacock.exe file shows me the terminal window as bellow.

<img width="721" alt="Screen Shot 2021-01-29 at 12 26 40 AM" src="https://user-images.githubusercontent.com/77819763/106159930-a97fa380-61c8-11eb-8038-f400e02e02fb.png">

Is there any solution for this?
Thanks.

**Reply by aeslaughter** [reply]
I wonder if this is the problem: https://keremkoseoglu.com/2020/11/14/pyqt5-problem-on-macos-big-sur-solved

Did you install the MOOSE environment with conda or manually?
If you are using conda, try searching for the available options and install one of the older ones. I am running 5.12.

```
conda search pyqt
conda install pyqt=5.12 # or whatever version you want to try out
```

**Reply by aeslaughter** [reply]
It seems that Peacock is not working on BigSur, I have two other reports of failure. I will need to investigate the problem, it will be next week at the earliest before I get to this.

**Reply by HeejaeJu** [reply]
Thanks for your help. I will wait for solution.
Have a nice weekend!
